README.md

```diff
@@ -88,7 +88,7 @@ To run and test the plugin in for development purpose, configure a local Nextflo
 The plugin can be tested without using a local Nextflow build using those steps:
 
 1. generate required artifacts with `make buildPlugins`
-2. copy build/plugins/your-plugin to `$HOME/.nextflow/plugins`
+2. copy build/plugins/your-plugin to `/.nextflow/plugins`
```

Comment:
Well, no, why copy in the root?

Reply:
I haven't changed anything but my plugin was only found under `/.nextflow/plugins`, but not under `$HOME/.nextflow/plugins`.

Reply:
It's a glitch in your env. Plugins are looked into `$HOME/.nextflow/plugins`

Reply:
I tried with the official Nextflow Docker container (`nextflow/nextflow:23.02.1-edge`)

Reply:
Well, in the container the $HOME may be root, but that's something specific to the container environment